src/Theme/ThemeNegotiator.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * @file
+ * Contains \Drupal\varbase_layout_builder\Theme\ThemeNegotiator
+ */
+namespace Drupal\varbase_layout_builder\Theme;
+
+use Drupal\Core\Routing\RouteMatchInterface;
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Theme\AjaxBasePageNegotiator;
+use Drupal\Core\Access\CsrfTokenGenerator;
+use Symfony\Component\HttpFoundation\RequestStack;
+
+
+class ThemeNegotiator extends AjaxBasePageNegotiator {
```

Comment:
Change class name to something specific and indicative to the module. For example: `VLBThemeNegotiator` Follow the same convention of other classes in the module.